### PR TITLE
Integrate conformance tests with new l2discovery and solver libs

### PR DESCRIPTION
**Pull request by edcdavid**
This PR integrates externalized l2discovery and simplegraph solver libraries into ptp conformance test:
- https://github.com/test-network-function/l2discovery-lib
- https://github.com/test-network-function/l2discovery-exports
- https://github.com/test-network-function/graphsolver-lib
- https://github.com/test-network-function/graphsolver-export
Also fixes a regression with BC scenario

### CODE REVIEW

File: test/utils/testconfig/testconfig.go [hunk 1]
```diff
@@ -225,39 +284,184 @@ func GetDesiredConfig(forceUpdate bool) TestConfig {
...
 	// Initialize desired ptp config for all configs
 	GetDesiredConfig(true)
 	// in multi node configuration create ptp configs
 	if GlobalConfig.PtpModeDesired != Discovery {
-		GlobalConfig.L2Config, err = l2discovery.GetL2DiscoveryConfig()
+		// Initialize l2 library
+		l2lib.GlobalL2DiscoveryConfig.SetL2Client(client.Client, client.Client.Config)
+
+		// Collect L2 info
+		config, err := l2lib.GlobalL2DiscoveryConfig.GetL2DiscoveryConfig()
```
Comment (edcdavid) [inline test/utils/testconfig/testconfig.go:295]: Need to add additional ptp capable filter in l2 discovery (was done here before) - in progress

Comment (edcdavid) [reply test/utils/testconfig/testconfig.go]: fixed

Comment (edcdavid) [reply test/utils/testconfig/testconfig.go]: fixed

**Review by aneeshkp — APPROVED** [review]

**Pull request merged**